src/Item_Project.php

```diff
@@ -265,16 +267,97 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
 
     public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $withtemplate = 0)
     {
+        /** @var array $CFG_GLPI */
+        global $CFG_GLPI;
 
         switch ($item->getType()) {
             case 'Project':
                 self::showForProject($item);
                 break;
 
             default:
-               // Not defined and used now
-               // Project::showListForItem($item);
+                if (in_array($item->getType(), $CFG_GLPI["project_asset_types"])) {
+                    self::showForAsset($item);
+                }
+                break;
         }
         return true;
     }
+
+    public static function showForAsset(CommonDBTM $item)
```

Comment:
This method wille be used only from the current class, please make it private.

```suggestion
    private static function showForAsset(CommonDBTM $item): void
```